test/test-scrape.bats

```diff
@@ -0,0 +1,16 @@
+setup() {
+    # get the containing directory of this file
+    # use $BATS_TEST_FILENAME instead of ${BASH_SOURCE[0]} or $0,
+    # as those will point to the bats executable's location or the preprocessed file respectively
+    DIR="$( cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd )"
+    # make executables in src/ visible to PATH
+    PATH="$DIR/../airootfs/usr/share/vx-img/:$PATH"
+}
+
+@test "can run our script" {
+    # notice the missing ./
+    # As we added src/ to $PATH, we can omit the relative path to `src/project.sh`.
```

Comment:
any reason to think that might be bad, as in it might pick up some other version of the `.sh` script and test that instead?

Reply:
I don't think so. Especially since we're sourcing it by name, unless somehow we get another `util.sh` or something I don't see how it would get confused?